opentreemap/exporter/tasks.py

```diff
@@ -161,12 +174,19 @@ def simple_async_csv(job_pk, qs):
 def custom_async_csv(csv_rows, job_pk, filename, fields):
     job = ExportJob.objects.get(pk=job_pk)
 
-    csv_obj = TemporaryFile()
-
-    writer = csv.DictWriter(csv_obj, fields)
-    writer.writeheader()
-    for row in csv_rows:
-        writer.writerow(sanitize_unicode_record(row))
+    # TODO: roll this into the @job_transaction pattern
+    # OR decide to leave separate and remove this TODO.
```

Comment:
I don't see any reason why you can't just use `@job_transaction` on this task too.  It doesn't seem any functionally different from whats being done here.